lib/model/model.go

```diff
@@ -1970,17 +1970,6 @@ func (m *Model) internalScanFolderSubdirs(ctx context.Context, folder string, su
 		UseWeakHashes:         weakhash.Enabled,
 	})
 
-	if err != nil {
-		// The error we get here is likely an OS level error, which might not be
-		// as readable as our health check errors. Check if we can get a health
-		// check error first, and use that if it's available.
-		if ferr := runner.CheckHealth(); ferr != nil {
-			err = ferr
-		}
-		runner.setError(err)
-		return err
-	}
```

Comment:
Should we not keep this given checkDir is going away?

Reply:
There could be another 
```golang
 	if err := runner.CheckHealth(); err != nil {
 		return err
 	}
```
here to abort a little bit earlier in case the scanner has scanned some stuff before the folder health problem arose.

Reply:
Scanning doesn't matter much, as there are periodic checks as the results arrive after the scan before stuff goes into the db.

Reply:
I should have been more explicit, this was what I meant: There was a health check before starting the scanner and this removed block of code is just polishing the error from now removed `checkDir`, so the only benefit of keeping a health check there is, that we may return slightly earlier instead of before sending to db or fchan was closed.

Reply:
Sure, yet I think there is still value to have one of the checks, as it seems we're axe'ing both.